### PR TITLE
Create CODE_OF_CONDUCT.md

### LINKED ISSUE #15: Create a CODE_OF_CONDUCT.md
Add a CODE_OF_CONDUCT.md file to set the ground rules for collaboration and communication within the project. This file should outline expected behaviors and provide guidelines for reporting violations.

### PR DESCRIPTION
**Pull request by santosomar**
Fixes #15

### CODE REVIEW

**Review by tschmidtb51 — COMMENTED** [review]
That sounds a lot like the one from http://contributor-covenant.org/version/1/4 (If so, I guess we should cite them as source.)

BTW: There are two other sections, that we use in [Secvisogram](https://github.com/secvisogram/secvisogram/blob/main/CONTRIBUTING.md#code-of-conduct)

**Comment by santosomar** [comment]
Thank you @tschmidtb51 ! I updated the file. This Code of Conduct is adapted from the [Contributor Covenant][homepage],
version 2.1, available at
[https://www.contributor-covenant.org/version/2/1/code_of_conduct.html][v2.1].

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,29 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (tschmidtb51) [inline CODE_OF_CONDUCT.md:63]: I guess we need to insert a contact method here...

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
I guess we need to add the contact info

**Pull request merged**